api/migrations/20200921110650_remove-auth-sessions.js

```diff
@@ -0,0 +1,24 @@
+exports.up = async knex => {
+  knex.schema.dropTable('auth_sessions');
+};
+
+exports.down = async knex => {
+  knex.schema.createTable('auth_sessions', table => {
+    table.comment('authentication sessions');
+    table
+      .string('session_id', 64)
+      .comment('randomly-generated session ID')
+      .notNullable();
+    table
+      .integer('user_id')
+      .notNullable()
+      .comment('user that owns the session');
+    table
+      .bigInteger('expiration')
+      .unsigned()
+      .notNullable()
+      .comment('when the session expires');
+
+    table.foreign('user_id').references('users.id');
+  });
+};
```

Comment:
Bonus points for the knex `down` migration! We have been bad about adding those. 😬